cookbooks/aws-parallelcluster-entrypoints/kitchen.entrypoints-config.yml

```diff
@@ -0,0 +1,14 @@
+---
+verifier:
+  name: inspec
+  inspec_tests:
+    - test
+
+suites:
+  - name: setup_envars
+    run_list:
+      - recipe[aws-parallelcluster-entrypoints::setup_envars]
```

Comment:
I don't see setup_envvars in the entrypoints cookbook, what am I missing?

Reply:
My mistake, will delete this file.